data/transactions/logic/eval_test.go

```diff
@@ -1563,7 +1563,8 @@ assert
 int 1
 `
 
-const testTxnProgramTextV8 = testTxnProgramTextV7
+const testTxnProgramTextV8 = testTxnProgramTextV7 + `
```

Comment:
`TestTxn()` is still passing for me even without the additional line. Is there another purpose?